tests/integration/components/project-list-test.js

```diff
@@ -493,4 +502,228 @@ module('Integration | Component | project list', function (hooks) {
 
     assert.dom('[data-test-project-list-header-clear-filter]').exists();
   });
+
+  test('It switches between card and list views', async function (assert) {
+    this.server.createList('project', 3);
+
+    this.server.get('/v3/projects', (schema) => {
+      const results = schema.projects.all().models;
+
+      return { count: results.length, next: null, previous: null, results };
+    });
+
+    this.server.get('/profiles/:id/unknown_analysis_status', (schema, req) => {
```

Comment:
Do we require this since we've migrated to the project's `showUnknownAnalysis` for this info?